docs/src/modules/brandingTheme.ts

```diff
@@ -476,18 +475,18 @@ export function getThemedComponents(): ThemeOptions {
         styleOverrides: {
           root: ({ theme, ownerState }) => ({
             transition: 'all 120ms ease-in',
-            '&:active': { transform: 'scale(.96)' },
```

Comment:
It feels like we are missing visual feedback for the active style. It doesn't have to be scale, but I think there should be something.